Cakefile

```diff
@@ -434,3 +435,17 @@ task 'test:browser', 'run the test suite against the merged browser script', ->
   (-> eval source).call result
   testResults = runTests result.CoffeeScript
   process.exit 1 unless testResults
+
+task 'test:webpack', 'run webpack test', ->
```

Comment:
Rather than this be its own Cake task, can you create a `test/integrations.coffee` file and put this code there? That’s where the other related tests can go, that test if the `coffeescript` module as `require`d by other Node projects still works.

The top of the file should have `return if global.testingBrowser` like you see at the top of `test/repl.coffee`, as obviously a test that calls `spawnNodeProcess` requires Node. You can attach `spawnNodeProcess` to `global` to make it available in the test file.